package.json

```
@@ -4,8 +4,8 @@
     "description": "INITIAL README",
     "main": "server.js",
     "scripts": {
-        "test": "test server.js",
-        "start": "node server.js",
+        "test": "mocha || true",
+        "start": "nodemon server.js",
```

```suggestion
        "start": "node server.js",
```

This has to remain as a node executable because this is what our server runs in production. We can't have nodemon running in production. If you need to run nodemon while developing see the command below `dev`. You can you `npm run dev`.